yb-voyager/cmd/config.go

```diff
@@ -598,64 +603,115 @@ func bindCobraFlagsToViper(cmd *cobra.Command, v *viper.Viper) ([]ConfigFlagOver
 				ConfigKey: configKey,
 				Value:     val,
 			})
-		} else if configKey = SourceDBConfigPrefix + strings.TrimPrefix(f.Name, SourceDBFlagPrefix); strings.HasPrefix(f.Name, SourceDBFlagPrefix) && v.IsSet(configKey) {
-			// Handle source db type flags
+		} else if slices.Contains(commandsUsingSourceReplicaConfig, commandNameKey) {
+			// Handle source-replica config flags
+			bindSourceReplicaFlags(cmd, v, f, commandNameKey, &bindErr, &overrides)
```

Comment:
nit: 
how about let it return the err and then: 
```
if err != nil {
    bindErr = err
    return
}
```

Just so that it's consistent with the other blocks of if-else. (in the ones above, we are immediately returning on bindErr, and in these blocks, we are not).

Reply:
Yep done